src/label_studio_sdk/converter/brush.py

```diff
@@ -175,10 +155,7 @@ def convert_task(item, out_dir, out_format="numpy"):
     """Task with multiple annotations to brush images, out_format = numpy | png"""
     for from_name, results in item["output"].items():
         save_brush_images_from_annotation(
-            item["id"],
-            item["annotation_id"],
-            item["completed_by"],
-            from_name,
+            os.path.basename(item["input"]["image"]),
```

Comment:
what if you have another `image` field name in the task.data?

Reply:
Sorry I'm not following what you mean.  When I loaded the value of the dictionary in item["input"] I get the following:
item["input"] = {'image': '/data/upload/2/c2825893-NathonFamous.png'}

Is there a better place to pull the image name?  I'm having troubles tracing back the definition of the input data.